iceoryx_examples/icedelivery/iox_subscriber_typed_modern.cpp

```diff
@@ -36,24 +36,21 @@ static void sigHandler(int f_sig [[gnu::unused]])
 void subscriberHandler(iox::popo::WaitSet& waitSet)
 {
     // run until interrupted
-    while(!killswitch)
+    while (!killswitch)
     {
         auto triggeredConditions = waitSet.wait();
-        for(auto& condition : triggeredConditions)
+        for (auto& condition : triggeredConditions)
         {
             auto subscriber = dynamic_cast<iox::popo::TypedSubscriber<Position>*>(condition);
-            if(subscriber)
+            if (subscriber)
             {
                 subscriber->take()
-                        .and_then([](iox::popo::Sample<const Position>& position){
-                            std::cout << "Got value: (" << position->x << ", " << position->y << ", " << position->z << ")" << std::endl;
-                        })
-                        .if_empty([]{
-                            std::cout << "Didn't get a value, but do something anyway." << std::endl;
-                        })
-                        .or_else([](iox::popo::ChunkReceiveError){
-                            std::cout << "Error receiving chunk." << std::endl;
-                        });
+                    .and_then([](iox::popo::Sample<const Position>& position) {
```

Comment:
Shouldn't we use `auto & position` here?

Reply:
Doesn't work in C++14 with const in this case. Same as `iox_subscriber_untyped_modern.cpp`.